help.html

```diff
@@ -22,4 +22,4 @@ <h2>Neurostars</h2>
 
 <h2>Github issues</h2>
 
-<p>Most of the projects have an issues page on which you are (very) welcome to report about problems that you have encountered using the software. For example, for issues related to the Dipy software library, please go here: <a href="https://github.com/nipy/dipy/issues" target="_blank">https://github.com/nipy/dipy/issues</a>. More informal conversations can be struck on the gitter pages of the projects (e.g. <a href="https://gitter.im/nipy/dipy" target="_blank">https://gitter.im/nipy/dipy)</a></p>.
+<p>Most of the projects have an issues page on which you are (very) welcome to report about problems that you have encountered using the software. For example, for issues related to the Dipy software library, please go here: <a href="https://github.com/dipy/dipy/issues" target="_blank">https://github.com/dipy/dipy/issues</a>. More informal conversations can be struck on the gitter pages of the projects (e.g. <a href="https://gitter.im/nipy/dipy" target="_blank">https://gitter.im/nipy/dipy)</a></p>.
```

Comment:
Is there still a gitter channel at nipy/dipy (see last line of this section) or does that need to be updated too?

Reply:
Actually I just checked and confirmed that there is - so this is correct! This LGTM, let's merge.

Reply:
That's right, the DIPY gitter channel still dwells under the NIPY umbrella.

Thanks for having taken care of the PR.

Reply:
Of course! Thanks for fixing the links.